apps/chat/src/store/conversations/conversations.epics.ts

```diff
@@ -2323,6 +2323,10 @@ const uploadConversationsWithFoldersRecursiveEpic: AppEpic = (
               FeatureType.Chat,
             );
 
+            const rootConversationId = conversations.toSorted(
+              (a, b) => a.folderId.length - b.folderId.length,
+            )[0].id;
```

Comment:
```suggestion
            const rootConversationId = conversations.filter(c => isRootConversationsId(c.id))[0].id;
```

Reply:
I've tried using isRootConversationsId, but it doesn't work in all cases here. Actually, I named this variable incorrectly, because it isn't the root conversation Id in all cases. It should be the Id of the conversation closest to the root folder. So, I've now renamed it as topLevelConversationId.